### ISSUE TITLE
[Bug] Not seeing a "Next Up" notification

### ISSUE THREAD

**Issue by tagnullde** [ironmonk88/monks-little-details#4]
Maybe I am missing the point - but the next player won't receive a notification for "Next Up" nor "Your Turn". 

What can I provide for better troubleshooting?

**Comment by ironmonk88** [comment]
I've heard some people are having issues with it.
What version of the module are you using?

**Comment by tagnullde** [comment]
1.0.4

**Comment by ironmonk88** [comment]
Is this still an issue?  The latest version is out now and fixed some things.

**Comment by tagnullde** [comment]
Yes. I've just updated. 

I had 2 PC Actors and 1 Monster Actor. While clicking "forward" to jump to the next combatant in the Tracker I can't see a on-screen-text or chat-notification.

**Comment by ironmonk88** [comment]
And you're viewing it as a player, not as a GM?  The notification will only show up for the player that controls the token.

**Comment by tagnullde** [comment]
Correct, as a Player in a separate Chromium Incognito window.

**Comment by ironmonk88** [comment]
Are there any error messages in the browser debug window?
The notification will only show if the user isn't a GM and the actor who's turn it is, is owned by the current player.
I'll look to add some more debugging information, but it probably won't show up until the next version release.
Apologies for it taking so long to track this one down.

**Comment by tagnullde** [comment]
There are, but to me - they don't seem related.

![image](https://user-images.githubusercontent.com/44159008/106497165-0081c300-64be-11eb-817b-5f025cd0ab57.png)

And no worries - it's just a minor thing I noticed. Glad someone takes the time to code mods like this in the first place.

**Comment by FatBiscuit** [comment]
I also had this issue where the notifications would not trigger plus the sounds would not trigger for each round, next turn, and current turn notifications.  Pressing m to teleport tokens was also not working.  I went through all the modules I have installed one-by-one and found that Maestro was causing some conflict with the way I had it configured.  Now the only thing I have Maestro doing is triggering sound effects for weapon strikes, and all Monk's Little Details features are working fine.  Thanks to Monk, this is a great module!

**Comment by ironmonk88** [comment]
Silly question, but has the combat been started?

**Comment by ironmonk88** [comment]
Pressing m to teleport should be fixed now.  Found the issue.

**Comment by tagnullde** [comment]
> Silly question, but has the combat been started?

Yes. ;)

**Comment by ironmonk88** [comment]
Lol.  I figured, but never hurts to ask.  :-)  Debug code is active in the latest version.  On the players machine, when the turn changes there should be a log in the console from monks-little-details that says "Check combat turn" and should list the current combatant's turn, the entity involved and then two boolean.  The first is if MLD thinks the current combatant is the active one, the second is if it thinks it's next up.  Can you run it through a combat round and let me know what it's reporting.

**Comment by FatBiscuit** [comment]
Latest version is working great as of today.  Thanks!

**Comment by tagnullde** [comment]
Hm. I don't see "Check combat turn". I did it anyways and got this:

![grafik](https://user-images.githubusercontent.com/44159008/107929550-142b3000-6f7a-11eb-994d-bb674681f729.png)

![grafik](https://user-images.githubusercontent.com/44159008/107929601-2d33e100-6f7a-11eb-904c-81824ccbfcbb.png)

I do hear the "sound" - but I expect a text-notification on screen. Am I right? Not that we chase ghosts here because I am dumb.
Also, screenshots have been made from the same machine in an incocnito mode window as the player.

**Comment by ironmonk88** [comment]
There should be a text notification.  Are you hearing the round sound or the next turn sound?
I've added a little more debugging information.
There should be a "checking combat started" in the console as well as "Check combat turn".

**Comment by tagnullde** [comment]
Check Combat Started:
![grafik](https://user-images.githubusercontent.com/44159008/108979830-5a277880-768b-11eb-8a02-56a12194cc17.png)

Check Combat Started 2:
![grafik](https://user-images.githubusercontent.com/44159008/108991203-138c4b00-7698-11eb-9108-a030db8e00fd.png)

Update Combat (after Check Combat Started 2)
![grafik](https://user-images.githubusercontent.com/44159008/108991276-3159b000-7698-11eb-8775-fe4b071d3dad.png)

Cycling through the Combat Tracker (1 PC, 1 NPC)
![grafik](https://user-images.githubusercontent.com/44159008/108991572-91e8ed00-7698-11eb-8532-a712f446b1de.png)

I don't see "check combat turn" though. Only Turn info I found was in the Update Combat:
![grafik](https://user-images.githubusercontent.com/44159008/108992013-13d91600-7699-11eb-9c74-ad9dc437d5cf.png)

Gamemaster Console Overview of 1 round of Combat :
![grafik](https://user-images.githubusercontent.com/44159008/108992229-5ac70b80-7699-11eb-9263-dd370cc1b320.png)

Player Console Overview of 1 round of Combat:
![grafik](https://user-images.githubusercontent.com/44159008/108992433-97930280-7699-11eb-86ed-07027d75052d.png)

I hear the round sounds. The one you also hear when combat starts.
Interesstingly enough I do see way more errors in the player console as in the gm console.

**Comment by ironmonk88** [comment]
I think I found it!  You must have another combat on a different scene, and my code was using that one instead of the currently active one.
I've updated the code to use the correct one now.  I'll run it through some testing and get another patch out soon.

Thank you so much for putting up with this lengthy debug process.

**Comment by tagnullde** [comment]
I double check all my scenes twices. And I indeed found one combat that I forgot the "end".
After doing so, I went back retesting but still no message on screen. What I also noticed - that I don't see the turn marker from the "next up" module most of the time. I'll disable all modules for the next test again. Might be a combination of multiple factors. Need to attend other things right now so don't have the time to do so. Will let you know what the outcome is.
